src/Reader/Http/Response.php

```diff
@@ -155,14 +157,16 @@ private function validateHeaders(array $headers)
 
     /**
      * Normalize header names to lowercase.
+     * Also ensures multi-value headers are represented as a single string, via
+     * comma concatenation.
      *
      * @return array
```

Comment:
Let's document this to be an `array<string>` then? Previously, it was an `array<string|array<string>>`, so this change looks like a breaking change 🤔

Reply:
Its the opposite I think. Before the `normalizeHeaders` was expecting `array<string, string>` and now it can handle `array<string, string|array<string>>`, as the logic that was in [LaminasHttpClientDecorator prepareResponseHeaders](https://github.com/laminas/laminas-feed/blob/2.23.x/src/Reader/Http/LaminasHttpClientDecorator.php#L106) now is included in Response `normalizeHeaders` (see `validateHeaders` in `Response` is not accepting multi value (L147 in my branch).

So I thought its not a breaking change as now it can handle what it was handling before and in addition multi-value headers?